addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -2407,6 +2410,14 @@ public Action timer_datacollect(Handle timer) // 1/5th Hz
 	return Plugin_Continue;
 }
 
+public Action timer_progress(Handle timer) // 30 sec
+{
+	if (!zf_bEnabled) return Plugin_Continue;
```

Comment:
Should return Plugin_Stop to actually stop repeat timer, same to other timers above